erpnext/regional/india/e_invoice/einvoice.js

```diff
@@ -17,19 +17,26 @@ erpnext.setup_einvoice_actions = (doctype) => {
 			};
 
 			if (!irn && !__unsaved) {
-				const action = () => {
-					frappe.call({
-						method: 'erpnext.regional.india.e_invoice.utils.get_einvoice',
-						args: { doctype, docname: name },
-						freeze: true,
-						callback: (res) => {
-							const einvoice = res.message;
-							show_einvoice_preview(frm, einvoice);
-						}
-					});
-				};
+				frappe.db.get_single_value("E Invoice Settings", "generate_irn_only_on_submission").then(val => {
```

Comment:
fetch this along with `'einvoicing_enabled'`

Reply:
Feature removed due to some validation issues